.gitignore

```diff
@@ -1,3 +1,3 @@
 node_modules
-components
-test/build.js
\ No newline at end of file
+dist
```

Comment:
`.npmignore` is parsed instead of this, so I have it include everything except `dist`